add more hints how to set the database_id (#2656)
cap-js/docs: guides/multitenancy/index.md

For SAP HANA TMS v2, you also need to specify the database ID of the database that you plan to use for your tenant containers. You can specify this using the [`cds.xt.DeploymentService` configuration](/@external/guides/multitenancy/mtxs#deployment-config).

> This is only the default database ID. If you want to specify different database IDs per tenant, you need to [add the database ID to the payload of the individual subscription using a handler](/@external/guides/multitenancy/mtxs#example-handler-for-saasprovisioningservice).

Reviewer: What is a "database ID per tenant"? Is that a container inside the database  I configured in `cds.xt.DeploymentService` configuration?

Author: it is the database on which the tenant HDI container will be created

> This is only the default database ID. If you want to specify different database IDs for a tenant HDI container, you need to [add the database ID to the payload of the individual subscription using a handler](/@external/guides/multitenancy/mtxs#example-handler-for-saasprovisioningservice).

Reviewer: ```suggestion
With `cds.xt.DeploymentService` you only configure the default database ID. If you want to specify different database IDs for a tenant HDI container, you need to [add the database ID to the payload of the individual subscription using a handler](/@external/guides/multitenancy/mtxs#example-handler-for-saasprovisioningservice).
```
